### ISSUE TITLE
Permanent uncontrolled channel action of all active channels at Version 1.2.8

### ISSUE THREAD

**Issue by Skippydi** [FrSkyRC/ETHOS-Feedback-Community#1704]
I flashed my X10s to the new version 1.2.8 with Isrm module. This error that you can see in the video often occurs after switching on, but also while programming. It affects all active channels. It is also passed to the servos.
After switching off and on again, the error is usually gone. I don't want to get it in flight though.I went back to version 1.2.7, the error has not (yet) occurred there.
This problem never occurs with OTX or EdgeTX
Hardware Frsky Horus X10s and ISRM Modul
Ethos 1.2.8

https://user-images.githubusercontent.com/68751162/170214742-2070a4da-1bd7-4acf-99e5-fd3c6f8c41f8.mp

**Comment by bsongis-frsky** [comment]
What are the mixes on those channels? Have you properly done the calibration of your sticks / pots?

**Comment by loetefix** [comment]
Looks like the sticks / pots are not calibrated

**Comment by pld74** [comment]
If calibration is recommended or needed after a software upgrade, may I suggest that it automatically goes into calibration post firmware upgrade?

**Comment by Skippydi** [comment]
Pots and sticks are calibrated immediately after flashing. Even after flashing AND calibrating again, the problem reappeared. Unfortunately I can't reproduce when it occurs. I'll try today to see if I can somehow reproduce the problem.

**Comment by Skippydi** [comment]
Here is the situation after switching on and calibrating

https://user-images.githubusercontent.com/68751162/170475024-cd305d91-6afe-43ca-8c58-413f4d35b6e4.mp4

**Comment by MIKE-DAILEY** [comment]
I've tried to replicate your issue using a X10S ISRM running 1.2.8. FCC and I cannot.  I powered it on and off ten times with no issues and have not seen any issues using the system.  If I was having your issues I would reflash the system bootloader and firmware with a fresh download, try a new default model and rebuild the SD card.

**Comment by Skippydi** [comment]
That's the first thing I do in such a case. I'm not bothering you with that.
The error occurred 3x today, 1x when switching to a different model.

**Comment by bsongis** [comment]
What is displayed for "Sticks" in System/Info? PWM or ADC sticks?

**Comment by Skippydi** [comment]
It shows ADC Sticks

**Comment by bsongis** [comment]
And what kind of sticks do you have?

**Comment by Skippydi** [comment]
This one:
MC12P all CNC digital higher accuracy ten bearings hall sensor gimbals and extendable stick ends

**Comment by bsongis-frsky** [comment]
https://www.frsky-rc.com/product/mc12p/
I don't have any of those sticks at hand, but it should be PWM sticks, no?

**Comment by Skippydi** [comment]
I think it doesn't matter if Hall sensors or potentiometers are used.  It should always be a PWM signal.

**Comment by bsongis-frsky** [comment]
The stick internal protocol can be either PWM or ADC, obviously your MC12P are detected as ADC when it is PWM
I don't have those sticks, I will ask for tests in FrSky

**Comment by Skippydi** [comment]
The same issue by my friend.
Frsky Horus X10s  Access

https://user-images.githubusercontent.com/68751162/172462726-923010f7-a66e-441d-8577-60c5233605e6.MOV

**Comment by bsongis** [comment]
@Skippydi same sticks?

**Comment by Skippydi** [comment]
Yes, same Sticks

**Comment by EngelMT** [comment]
One off our customer had this issue some weeks before. We flasht complete new and spend him a new SD Card and all works fine. After 2 weeks the issue comes back. We changed the main board and all was fine again. But after some days the issue comes back.

Never in flight. Only on the ground after switch on the radio. Last but not least we changed him the radio. Now it seems to work well for several weeks.

We never thought of an ETHOS problem. But apparently it only occurs with ETHOS.
